tests/browser_context_options_test.go

```diff
@@ -60,7 +60,11 @@ func TestBrowserContextOptionsSetViewport(t *testing.T) {
 		},
 	}))
 	require.NoError(t, err)
-	t.Cleanup(bctx.Close)
+	t.Cleanup(func() {
+		if err := bctx.Close(); err != nil {
```

Comment:
We don't want to fail to test for a closing error here, but it's still a good idea to log the error to diagnose any possible issues.